src/share/poudriere/common.sh

```diff
@@ -1241,6 +1241,7 @@ run_hook_file() {
 		    SETNAME="${SETNAME-}" \
 		    PACKAGES="${PACKAGES-}" \
 		    PACKAGES_ROOT="${PACKAGES_ROOT-}" \
+		    VERBOSE="${VERBOSE-}" \
```

Comment:
Lets go with a default of 0 here? Everywhere else it is `${VERBOSE:-0}`.
It's probably not blank anyway.
```
src/share/poudriere/common.sh:: ${VERBOSE:=0}
```